src/Features/Traits/Optional/SpatiePermission.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits\Optional;
+
+use Behat\Gherkin\Node\TableNode;
+use Blumilk\BLT\Features\Traits\Application;
+use Blumilk\BLT\Helpers\ContextHelper;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use PHPUnit\Framework\Assert;
+
+trait SpatiePermission
+{
+    use Application;
+
+    /**
+     * @When :objectName can have role
+     * @When user can have role
+     * @throws BindingResolutionException
+     */
+    public function objectCanHaveRoles(string $objectName = "User"): void
+    {
+        $objectClass = ContextHelper::getClassHelper()->recognizeObjectClass($objectName);
+        $used_traits = class_uses_recursive($this->getContainer()->make($objectClass));
+        Assert::assertTrue(in_array("Spatie\Permission\Traits\HasRoles", $used_traits, true));
```

Comment:
```suggestion
        $usedTraits = class_uses_recursive($this->getContainer()->make($objectClass));
        Assert::assertTrue(in_array("Spatie\Permission\Traits\HasRoles", $usedTraits, strict: true));
```

Reply:
Plus you could move this class reference to some other file like here: https://github.com/blumilksoftware/blt/blob/main/src/LaravelContracts.php